src/index.html

```diff
@@ -17,9 +17,21 @@
 
 <body class="layout_frontend">
     <app-root>
+        <script>
+            (function () {
+                // Set theme before app loads - prevents flash of white when using dark mode
+                let theme = window.localStorage.getItem('theme')?.replace(/"/g, '');
```

Comment:
This was a design decision discussed with Danielle. Concerns were:
* surprising existing users who wake up one day to find their web vault in dark mode
* new users being presented with dark colours on their first interaction with Bitwarden, rather than our official brand colours
* the marketing campaign registration page not being designed with dark theme in mind - this will require some coordination with marketing, and since there will be some registration improvement work on the 2022 roadmap, we'll revisit it at that time

I know this makes it slightly out of step with other clients (which default to `system`), but I don't think that's a big deal.

Reply:
Fair enough, do we have a card in Asana to track this?

Reply:
I'll follow up with Danielle and make sure it's tracked

Reply:
https://app.asana.com/0/1153292148278596/1201082114808085/f